lib/ur/contract/contract.go

```diff
@@ -237,6 +241,8 @@ func (r *Report) FieldPointers() []interface{} {
 		&r.FolderUsesV3.AlwaysWeakHash, &r.FolderUsesV3.CustomWeakHashThreshold,
 		&r.FolderUsesV3.FsWatcherEnabled,
 
+		&r.DeviceUsesV3.Untrusted,
```

Comment:
This is not needed, but if you're adding it, you should also add the column name,

Reply:
Ah yeah right, lazily searched for the wrong term (FolderUsesV3).instead of trying to understand. Why not necessary though?

Reply:
This was used for reading columns, and inserting into json tables. The migration is done, we don't have the data that was behind those columns.

Reply:
Removed it. Maybe that code should be pruned or moved to a file with "legacy" in its name just to make it clear?

Reply:
Agree, we can delete the code given the migration is done, I guess unless someone builds from source, or does a major version jump on their own UR server.